schema/deploy/shipping/views.sql

```diff
@@ -3283,10 +3286,43 @@ create or replace view shipping.__uw_priority_queue_v1 as (
         -- CHR 97 is 'a'
         -- DOW is 0 through 6, starting on Sunday; therefore, Sunday is group 'a'
         where uw_housing_group = CHR(EXTRACT(DOW from current_date)::int + 97)
+        -- Filter to participants who are not in an added surveillance group becuase they already get invited weekly
+        and added_surveillance_groups is null
```

Comment:
Is it possible for a participant to be (accidentally) marked for both `uw_housing_group` and `added_surveillance_groups` in REDCap? If that's the case, then that PT would be excluded from both groups in this priority queue...

I'm not sure if this is something we can guard against in REDCap. Is this something we can guard/alert on in the [ETL update](https://github.com/seattleflu/id3c-customizations/pull/202)?

Reply:
I've resisted putting this kind of check into the ETL because I didn't want to deal with more warnings in  id3c-alerts. But we say that the value of our ETLs is the data checks that we do inside them, so that is the right place to put this check.
I put that into Trello card https://trello.com/c/OFqlR28R
It's quick work, so I stuck it into Next Up. It won't block this SQL change but it would be good to have done soon.